sdk/servicebus/Azure.Messaging.ServiceBus/tests/Properties/AssemblyInfo.cs

```diff
@@ -4,4 +4,3 @@
 using NUnit.Framework;
 
 [assembly: Parallelizable(ParallelScope.All)]
-[assembly: NUnit.Framework.Timeout(5 * 60 * 1000)]
```

Comment:
Your change greatly reduces this;  5 seconds is not long enough for the Live tests and probably too short for some of the background operations for the Processor as well.

Reply:
Live tests will be addressed as part of this PR, or I'll abandon this approach. 

For the other tests, we can just add a more local attribute on those specific tests.

Reply:
I'd prefer to opt-out at the service directory level for Event Hubs.   I'll let Josh make the call for Service Bus.

Reply:
The intent of this change is to avoid long running unit tests. Wouldn't opt in be better (it should be an exception and very intentional to check in a long running test)

Reply:
As long as this timeout doesn't apply to live tests, I think it should be okay.

Reply:
revert?